docs/source/config/details.rst

```diff
@@ -111,7 +111,7 @@ You can see the Pygments styles available on your system by running::
     import pygments
     list(pygments.styles.get_all_styles())
 
-Additionally, ``TerminalInteractiveShell.highlighting_style_overrides`` can override
+Additionally, ``Terminal.InteractiveShell.highlighting_style_overrides`` can override
```

Comment:
I'm Thanks, but I'm pretty at least some of these are correct: 

```
In [1]: from IPython.terminal.interactiveshell import TerminalInteractiveShell, InteractiveShell

In [2]: TerminalInteractiveShell.highlighting_style_overrides
Out[2]: <traitlets.traitlets.Dict at 0x105691eb8>

In [3]: InteractiveShell.highlighting_style_overrides
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-3-e268df97a2b4> in <module>
----> 1 InteractiveShell.highlighting_style_overrides

AttributeError: type object 'InteractiveShell' has no attribute 'highlighting_style_overrides'
```

note that there is __both__ TerminalInteractiveShell and InteractiveShell, the former one used in terminal IPython, the later one inherited from in IPykernel to connect to – for example – the notebook.